setup.py

```diff
@@ -135,5 +171,11 @@ def main():
         print(hilite(msg, ok=False), file=sys.stderr)
 
 
+if sys.version_info[0] < 3:  # noqa: UP036
+    sys.exit(
+        'Python 2 is no longer supported. Latest version is 1.5.10; use:\n'
+        'python3 -m pip install pyftpdlib==1.5.10'
+    )
```

Comment:
Is the mention of `python3` here a typo?

Reply:
It is, thanks. Fixed in c4ceda49.